src/main/java/com/gamegoo/service/member/AuthService.java

```diff
@@ -127,10 +128,10 @@ public Member joinMember(String email, String password, String gameName, String
      * @param email
      */
     @Transactional
-    public void sendEmail(String email) {
+    public void sendEmail(String email, Boolean ischeck) {
```

Comment:
이 부분  isCheck 으로 변경하면 좋을 것 같습니다
```suggestion
    public void sendEmail(String email, Boolean isCheck) { 
```

Reply:
넵! 변경하겠습니다!